tests/test_api.py

```diff
@@ -111,10 +111,10 @@ def read(self):
 
 
 def test_api_str():
-    api = API('localhost', 8126)
-    assert str(api) == 'localhost:8126'
+    api = API('localhost', 8126, https=True)
+    assert str(api) == 'https://localhost:8126'
     api = API('localhost', 8126, '/path/to/uds')
-    assert str(api) == '/path/to/uds'
+    assert str(api) == 'http:///path/to/uds'
```

Comment:
This should be `unix:///path/to/uds`?

Reply:
that was my initial plan, but then how do you signify HTTPS + UDS? should it just not be allowed?

Reply:
https means HTTP over TLS. AFAIK, TLS runs normally on TCP.
So I don't think we should care about Unix Domain Socket + TLS.

By the way, `unix:///path/to/uds` is not accurate. In fact, we speak HTTP over Unix Domain Socket. So we should probably write `http+unix` or something.
`unix:///path/to/uds` is an abbreviated notation.

Reply:
I just want to make sure that this PR is consistent with https://github.com/DataDog/dd-trace-py/pull/1054.

Reply:
ok. I went with `unix://` for both HTTP and HTTPS